packages/headless/src/features/analytics/index.ts

```diff
@@ -38,14 +38,50 @@ import {
   logFacetUpdateSort as logFacetUpdateSortAlias,
   logFacetBreadcrumb as logFacetBreadcrumbAlias,
 } from '../facets/facet-set/facet-set-analytics-actions';
+export {
+  FacetValueAnalyticPayload,
+  FacetUpdateSortAnalyticPayload,
+} from '../facets/facet-set/facet-set-analytics-actions';
```

Comment:
Without the `export import` statement, moving these types inside `FacetAnalyticsActions` is complicated. It seems like they belong outside the namespace anyway.